README.md

```diff
@@ -41,45 +40,12 @@ You can also install the bleeding edge version of `hssm` directly from this repo
 ```
 pip install git+https://github.com/lnccbrown/HSSM.git
 ```
-**Note**: Possible solutions to any issues with installations with hssm can be located [here](https://github.com/lnccbrown/HSSM/discussions). We recommend leveraging an environment manager with Python 3.9~3.11 to prevent any problems with dependencies during the installation process. Please note that hssm is tested for python 3.9, 3.10, 3.11. Use other python versions with caution. For more detailed guidance, please refer to this [discussion](https://github.com/lnccbrown/HSSM/discussions/152). 
-
-### Optional Installation
-
-**Dependency for graph() Function**
-
-Note: In addition to the installation of the main hssm class, there is an optional dependency for the graph() function. This dependency requires graphviz, which can be installed conveniently using conda with the following command:
-
-```
-conda install -c conda-forge python-graphviz
-```
-
-Alternatively, you have the option to install the graphviz binaries manually and then install the Python bindings using pip with the following command:
-
-```
-pip install graphviz
-```
-
-**Dependency for sampler="nuts_numpyro"**
 
-To utilize the nuts_numpyro sampler, please follow these steps:
+You will need optional dependencies to use JAX-based samplers and graph the models.
```

Comment:
here is says optional, but I think the other pr said it's now a strict dependency?

Reply:
Yeah at the point of this PR numpyro was still optional. If we merge this first and the next one next, all will be updated just fine ;)